lib/notifications/client/response_notification.rb

```diff
@@ -7,6 +7,7 @@ class ResponseNotification
       content
       template
       uri
+      postage
```

Comment:
If postage is being returned as part of a `ResponseNotification` (as well as part of the new `ResponsePrecompiledLetter` class) then I think we need to add this to the docs in the responses for sending emails, texts and templated letters, e.g. here https://docs.notifications.service.gov.uk/ruby.html#response

Reply:
oops, I thought I deleted postage from standard response, I will do it now, sorry!